modules/mco-update-boot-images-about.adoc

```diff
@@ -7,27 +7,25 @@
 [id="mco-update-boot-images_{context}"]
 = About boot image management
 
-By default, for {gcp-first} and {aws-first} clusters, the Machine Config Operator (MCO) updates the boot image in the machine sets in your cluster whenever you update your cluster.
+[role="_abstract"]
+With boot image management enabled, the Machine Config Operator (MCO) manages and updates the {op-system-first} version of the boot image in the machine sets for your control plane or worker nodes. This means that the MCO updates the boot image whenever you update your cluster. Without boot image management enabled, if your cluster was originally created with an older {product-title} version, the boot image that the MCO would use to create new nodes is an older {op-system-first} version, even if your cluster is at a later {product-title} version. 
```

Comment:
This is a bit long for a shortdesc. I think you can break it here and it would be pretty solid
```suggestion
With boot image management enabled, the Machine Config Operator (MCO) manages and updates the {op-system-first} version of the boot image in the machine sets for your control plane or worker nodes. This means that the MCO updates the boot image whenever you update your cluster. 

Without boot image management enabled, if your cluster was originally created with an older {product-title} version, the boot image that the MCO would use to create new nodes is an older {op-system-first} version, even if your cluster is at a later {product-title} version. 
```